qrenderdoc/Windows/TextureViewer.ui

```diff
@@ -656,6 +689,9 @@
     </item>
     <item>
      <widget class="QToolButton" name="channelRed">
+      <property name="toolTip">
+       <string>Show Red</string>
```

Comment:
right click is not implemented. I believe it's better if this hint is added after the implementation is there